lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -26,15 +26,15 @@ class SynchronizeLdm < BaseAction
 
         description 'Specifies whether to transfer computed attributes'
         param :include_computed_attributes, instance_of(Type::BooleanType), required: false, default: true
+
+        description 'Allows to have facts with higher precision decimals'
+        param :exclude_fact_rule, instance_of(Type::BooleanType), required: false, default: false
       end
 
       class << self
         def call(params)
-          # set default value for include_computed_attributes
-          # (we won't have to do this after TMA-690)
-          include_ca = params.include_computed_attributes
-          include_ca = true if include_ca.nil?
-          include_ca = include_ca.to_b
+          include_ca = params.include_computed_attributes.to_b
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)